CryptoExchange.Net/Converters/BaseConverter.cs

```diff
@@ -71,7 +71,7 @@ public override bool CanConvert(Type objectType)
 
         private bool GetValue(string value, out T result)
         {
-            var mapping = Mapping.FirstOrDefault(kv => kv.Value.Equals(value, StringComparison.InvariantCultureIgnoreCase));
```

Comment:
I would suggest first trying to look up case sensitive and if nothing is found then try looking up case insensitive. So something like:
````C#
var mapping = Mapping.FirstOrDefault(kv => kv.Value.Equals(value, StringComparison.InvariantCulture));
if(mapping.Equals(default(KeyValuePair<T, string>)))
    mapping = Mapping.FirstOrDefault(kv => kv.Value.Equals(value, StringComparison.InvariantCultureIgnoreCase));
````

Reply:
Awesome idea! I've pushed a new commit with this change.